### PR TITLE
fix github action for release script

### PR DESCRIPTION
**Pull request by etsai-stripe**
### Reviewers
r? @
cc @stripe/developer-products

 ### Summary

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -47,7 +47,7 @@ jobs:
         uses: goreleaser/goreleaser-action@v2
         with:
           version: v0.184.0
-          args: release -f .goreleaser/linux.yml --rm-dist
+          args: release -f .goreleaser/linux.yml --rm-dist --debug
```
Comment (suz-stripe) [inline .github/workflows/release.yml:50]: might want to remove debug output in prod

File: scripts/publish-to-artifactory.sh [hunk 2]
```diff
@@ -13,37 +13,49 @@ REPO="stripe-cli-$FORMAT"
 PACKAGE="stripe"
 DISTRIBUTIONS="stable"
 COMPONENTS="main"
-DEBIAN_META=""
-UPLOAD_URL=""
+
+if [ -z "$ARTIFACTORY_USERNAME" ]; then
+  echo "ARTIFACTORY_USERNAME is not set"
+  exit 1
+fi
```
Comment (suz-stripe) [inline scripts/publish-to-artifactory.sh:20]: we can remove this

**Review by suz-stripe — COMMENTED** [review]
this looks fine apart from a couple of small changes 😄

File: scripts/publish-to-artifactory.sh (hunk 2 above)
Comment (etsai-stripe) [reply scripts/publish-to-artifactory.sh:20]: 🤦‍♀️

**Review by suz-stripe — APPROVED** [review]
LGTM 👍

File: scripts/publish-to-artifactory.sh (hunk 2 above)
Comment (Cordelro86) [reply scripts/publish-to-artifactory.sh]: Pay out my money to my account

File: .github/workflows/release.yml (hunk 1 above)
Comment (Cordelro86) [reply .github/workflows/release.yml]: When

**Pull request merged**